src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -26,7 +27,7 @@
 
         // Index Motor Speeds
 
-        public static double INDEX_FEEDER_SPEED = 0.15;
+        public static double INDEX_FEEDER_SPEED = 0.2;
         public static double INDEX_IN_SPEED = 0.35;
```

Comment:
The index first stage has to run at 1/2 of the speed of the 2nd stage of intake to avoid balls popping out. Please have this reflect the change in the intake in speed, unless this has been tested on the bot

Reply:
L + didnt ask + dont care + we dont know what motor is which so we can tune later + its alex btw + ratio

Reply:
wdym you don't know which motor is which????? You're changing the entirety of intake rn so like bruh + just change the variable + it takes you less than a 1/100th of a rat